create/how_to/search_bar_for_docs.md

```diff
@@ -114,7 +114,7 @@ If you don't want to use Docker, here are [other ways to run the scraper](https:
 `<absolute-path-to-your-config-file>` should be the **absolute** path of your configuration file defined at [the previous step](#configuration-file).
 
 The API key you must provide should have the permissions to add documents into your MeiliSearch instance. In a production environment, we recommend providing the private key instead of the master key, as it is safer and it has enough permissions to perform such requests.
```

Comment:
```suggestion
The API key you must provide should have the permissions to add documents into your MeiliSearch instance. In a production environment, we recommend providing the built-in Default Admin API key instead of the master key, as it is safer and it has enough permissions to perform such requests.
```

Reply:
Same as the ones below.

```suggestion
The API key you must provide should have the permissions to add documents into your MeiliSearch instance. We strongly recommend using an API key with limited permissions and a set expiry date whenever possible.
```

Reply:
Good point. I'll mark these as resolved in this PR.